From 77e82516a69ad7dd10ada425930ba35e933abcfb Mon Sep 17 00:00:00 2001
From: Ping Cheng <pinglinux@gmail.com>
Date: Tue, 4 Oct 2011 23:50:42 -0700
Subject: [PATCH] Input: wacom - don't expose LED inactive option

The LED also indicates the status of the tablet. Don't turn it off.

Reviewed-by: Eduard Hasenleithner <eduard@hasenleithner.at>
Tested-by: Eduard Hasenleithner <eduard@hasenleithner.at>
Signed-off-by: Ping Cheng <pingc@wacom.com>
Signed-off-by: Dmitry Torokhov <dtor@mail.ru>
---
 drivers/input/tablet/wacom.h                 |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/input/tablet/wacom.h b/drivers/input/tablet/wacom.h
index 00332d66..561f107 100644
--- a/drivers/input/tablet/wacom.h
+++ b/drivers/input/tablet/wacom.h
@@ -115,9 +115,9 @@ struct wacom {
 	bool open;
 	char phys[32];
 	struct wacom_led {
-		u8 select; /* status led selector (0..3, -1=none) */
-		u8 llv;    /* status led brightness no button */
-		u8 hlv;    /* status led brightness button pressed */
+		u8 select; /* status led selector (0..3) */
+		u8 llv;    /* status led brightness no button (1..127) */
+		u8 hlv;    /* status led brightness button pressed (1..127) */
 		u8 img_lum;   /* OLED matrix display brightness */
 	} led;
 };
-- 
1.7.6.2

